src/main/java/brave/kafka/interceptor/TracingConfiguration.java

```diff
@@ -78,4 +80,19 @@ String getStringOrDefault(String configKey, String defaultValue) {
   String getString(String configKey) {
     return getStringOrDefault(configKey, null);
   }
+
+  String getStringOrStringList(String configKey) {
```

Comment:
@dvillaj not sure I follow why this validation is required. Could you add a comment on why this may be needed?

Reply:
Hi @jeqo, I noticed that some properties, like bootstraps servers, can be a List of string, instead a single string, so I added this method to retrieve a property as String, no matter its type

Reply:
Sure. Though most of the time many bootstrap servers are passed as comma-separated string.

Could you add a comment on top of method with the reasoning and an example of passing bootstrap servers as a list?

Reply:
Sure